### PR TITLE
VB-1842 Add server side endpoint to bring back individual session inf…

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

It adds new end point to allow session schedule information to be returned for a particular day

## What is the intent behind these changes?

to allow the staff member to see the details of the visit time table

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt [hunk 1]
```diff
@@ -277,4 +275,58 @@ class SessionService(
...
+      return SessionTemplateFrequency.BI_WEEKLY
+    }
+    return SessionTemplateFrequency.WEEKLY
+  }
+
+  private fun isNotMoreThanAWeek(
+    firstSessionDate: LocalDate,
+    sessionTemplate: SessionTemplate
+  ): Boolean {
+    return sessionTemplate.validToDate != null && DAYS.between(firstSessionDate, sessionTemplate.validToDate) < 8
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt:323]: should this be < 7 for it to be one off?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt:323]: I think it has to be < 8 e.g.
MON (1), TUS (2), WEN (3), THU (4), FRI(5), SAT(6), SUN(7), MON(8) therefore less than 8 I think right?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt:323]: I'm being silly it's MON (0), TUS (1), WEN (2), THU (3), FRI(4), SAT(5), SUN(6), MON(7)

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by tpmcgowan — COMMENTED** [review]
LGTM!

**Pull request merged**